exchanges/kline/types.go

```diff
@@ -0,0 +1,44 @@
+package kline
+
+import (
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+)
+
+// Consts here define basic time intervals
+const (
+	OneMin     = time.Minute
+	ThreeMin   = 3 * time.Minute
+	FiveMin    = 5 * time.Minute
+	FifteenMin = 15 * time.Minute
+	ThirtyMin  = 30 * time.Minute
+	OneHour    = 1 * time.Hour
+	TwoHour    = 2 * time.Hour
+	FourHour   = 4 * time.Hour
+	SixHour    = 6 * time.Hour
+	TwelveHour = 12 * time.Hour
+	OneDay     = time.Hour * 24
```

Comment:
Can we make these consistent of value * time.Duration
```suggestion
	OneDay     = 24 * time.Hour
```

Reply:
Terrible coding practice; will fix.